internal/gather/cni.go

```diff
@@ -135,26 +134,49 @@ func gatherOVNResources(info *Info, networkPlugin string) {
 		return
 	}
 
-	// we check two different labels because OpenShift deploys with a different
-	// label compared to ovn-kubernetes upstream
-	ovnMasterpods, err := findPods(info.ClientProducer.ForKubernetes(), ovnMasterPodLabelOCP)
-	if err != nil || ovnMasterpods == nil || len(ovnMasterpods.Items) == 0 {
-		ovnMasterpods, err = findPods(info.ClientProducer.ForKubernetes(), ovnMasterPodLabelGeneric)
-		if err != nil {
-			info.Status.Failure("Failed to gather any OVN master ovnMasterpods: " + err.Error())
-		} else if ovnMasterpods == nil || len(ovnMasterpods.Items) == 0 {
-			info.Status.Failure("Failed to find any OVN master ovnMasterpods")
+	ovnKubePod := getOVNCmdsPod(info)
+	if ovnKubePod != nil {
```

Comment:
Also, I think we should capture the OVN DB details from EVERY zone in an OVN-IC setup. 
So, if each node is part of a separate zone, then we will have to collect details from every app=ovnkube-node. @aswinsuryan what do you say?

Reply:
Agree. But it should be a separate PR as focus of this one is to fix #910 I removed call to `submariner_route` coz it was causing error message in IC deployments.

Reply:
Even if the focus of this PR is to fix the crash. Lets not forget to collect the necessary logs as part of subctl 0.16.0 image. Otherwise, we will not have sufficient information while troubleshooting any issues with OVN IC. So we have to enhance/fix it before the official 0.16.0 release.

Reply:
Yes every kube-node will have a separate DB when we deploy a zone per node model and we need to capture from each. And for more than one zone per node, one of the ovn-kube pod will have the db in each zone.

Reply:
I'll update it to run command in each node for IC deployments, for now. It will lead to some duplication of data in global zone use case, but at least we won't miss on any information. We can revisit this logic later as part of other refactors.